tests/meta/archive/test_archive_api.py

```diff
@@ -348,7 +348,10 @@ def test_retrieve_samples(
         "api_request_from_content",
         return_value=ok_miria_response,
     ) as mock_request_submitter:
-        spring_archive_api.retrieve_samples([sample_with_spring_file])
```

Comment:
then I guess the fixture `sample_with_spring_file`is not used anymore?

Reply:
It is used in the next line 😉